server/views/icons/other/location.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 14 20"><circle class="icon__shape" fill="#1D1D1B" cx="7" cy="7" r="2"/><path class="icon__shape" fill="#1d1d1d" d="M7,5.7c0.7,0,1.3,0.6,1.3,1.3S7.7,8.3,7,8.3S5.7,7.7,5.7,7S6.3,5.7,7,5.7 M7,5C5.9,5,5,5.9,5,7s0.9,2,2,2s2-0.9,2-2S8.1,5,7,5L7,5z"/><path class="icon__shape" fill="#1d1d1d" d="M7,1c3.3,0,6,2.7,6,6c0,3.5-4.5,11.4-6,12c-0.4-0.2-1.8-1.6-3.5-4.8C2,11.4,1,8.5,1,7C1,3.7,3.7,1,7,1 M7,0C3.1,0,0,3.1,0,7s5,13,7,13c2,0,7-9.1,7-13S10.9,0,7,0L7,0z"/></svg>

```

Comment:
If the `icon__shape` is something we require - we should put it in the optimiser.